### PR TITLE
Syncer: enum serialization

### PR DESCRIPTION
**Pull request by TheCharlatan**

### CODE REVIEW

File: src/syncer.rs [hunk 1]
```diff
@@ -405,3 +453,97 @@ pub enum Event {
     TransactionBroadcasted(TransactionBroadcasted),
     TaskAborted(TaskAborted),
 }
+
+impl fmt::Display for Event {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "task")
```
Comment (h4sh3d) [inline src/syncer.rs:459]: This should be 'event'

File: src/syncer.rs [hunk 2]
```diff
@@ -229,6 +229,54 @@ pub enum Task {
     BroadcastTransaction(BroadcastTransaction),
 }
 
+impl fmt::Display for Task {
```
Comment (h4sh3d) [inline src/syncer.rs:232]: If you want to implement a deeper `Display` you can use the derive macro (we already depend on it) like here: https://github.com/farcaster-project/farcaster-core/blob/main/src/bundle.rs#L534-L535. It is part of `amplify` or something similar and there is nice derivation for Enums.

**Review by h4sh3d — APPROVED** [review]

File: src/syncer.rs (hunk 1 above)
Comment (TheCharlatan) [reply src/syncer.rs]: Good catch!

File: src/syncer.rs (hunk 2 above)
Comment (TheCharlatan) [reply src/syncer.rs]: Changed it to use the macro.

**Pull request merged**